src/components/Arrow/Button.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable react/destructuring-assignment */
+import * as React from "react";
+import { ArrowDirection, ArrowType, ButtonIconState } from "./Utils";
+import styles from "./Button.module.scss";
+import { createCompletedIcon, createEditIcon, createNoteIcon } from "./Icons";
+
+const renderIcon = (state: ButtonIconState, iconColor: string): JSX.Element => {
+  switch (state) {
+    case ButtonIconState.Completed:
+      return createCompletedIcon(iconColor);
+    case ButtonIconState.Notes:
+      return createNoteIcon(iconColor);
+    default:
+      return createEditIcon(iconColor);
+  }
+};
+
+export function clickArrow(): void {
+  // TODO
+}
+
+interface ArrowButtonProps {
+  arrowColor: string;
+  circleColor: string;
+  iconColor: string;
+  type: ArrowType;
+  direction: ArrowDirection;
+  buttonState: ButtonIconState;
+}
+
+export const ArrowButton: React.FC<ArrowButtonProps> = ({
+  arrowColor,
+  circleColor,
+  iconColor,
+  type,
+  direction,
+  buttonState,
+}): JSX.Element => {
+  let classNames = `${styles.button}`;
+
+  if (type !== ArrowType.Directional)
+    classNames += ` ${styles.buttonNonDirectional}`;
+  else classNames += ` ${styles.buttonDirectional}`;
```

Comment:
When using if/else, don't start with negation.

```suggestion
  if (type === ArrowType.Directional)
    classNames += ` ${styles.buttonDirectional}`;
  else classNames += ` ${styles.buttonNonDirectional}`;
```